### ISSUE TITLE
Set up SonarQube and triage initial issues

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#662]
# TL;DR
Add automated code scanning to increase confidence in security of our code.

# Narrative
As a developer, I would like to take advantage of automated code scanning tools as an extra pair of eyes to catch any security issues introduced.

Possibilities:
- SonarQube (would require integration with GitHub, would be in CI/CD pipeline by nature; could be used manually/on schedule)
- Bandit (Python tool, could potentially be integrated into precommit)
- Checking for unneeded requirements 

<details>
<summary>Feature Request fields</summary>
<p>
**With this `details` wrapper, you can just use this ticket to record your findings.
When you're ready to turn this into an implementation request, delete the scoping label and
this wrapper, leaving only the filled out information.**

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Need to select a tool and investigate advantages/disadvantages/setup steps. SonarQube and Github's CodeQL Analysis look like good tools to consider.

</p>
</details>

**Comment by mekline** [comment]
Spring cleaning! @okaycj , does SonarCloud replace/duplicate this? (AKA, can this issue be closed)

**Comment by okaycj** [comment]
> Spring cleaning! @okaycj , does SonarCloud replace/duplicate this? (AKA, can this issue be closed)

Yes it does.  We can close this.  I thought I would take a larger block of time to write up/resolve the issues brought up by sonarcloud.  Instead, I've been slowly working on the more "severe" issues.  

Closing.
